lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -2,3 +2,52 @@
 <section>
     {{!-- Fill out this registration form. --}}
 </section>
+<Registries::DraftRegistrationManager
+    @currentPage={{this.model.pageIndex}}
+    @modelTaskInstance={{this.model.taskInstance}}
+    @updateRoute={{route-action 'updateRoute'}}
+    as |draftRegistrationManager|
+>
+    <OsfLayout 
+        @backgroundClass='{{local-class 'ContentBackground'}}'
+        as |layout|>
+        <layout.leftNav as |leftNav|>
+            {{#each draftRegistrationManager.pageManagers as |pageManager index|}}
+                {{#let (hash
+                    active=(eq draftRegistrationManager.currentPage index)
+                    visited=pageManager.isVisited
+                    valid=pageManager.isValid
+                ) as |ctx|}}
+                    <leftNav.link
+                        data-analytics-name='Overview'
+                        @models={{array draftRegistrationManager.draftId}}
+                        @href='/drafts/{{draftRegistrationManager.draftId}}/{{index}}/'
+                        @icon={{if ctx.active 'circle-o' (if ctx.visited (if ctx.valid 'check-circle-o' 'exclamation-circle') 'circle') }}
```

Comment:
Does it make sense to add `icon` as a computed property on `pageManager`? Nested conditionals like this in templates is a bit confusing.